package com.codingsrv.RoleBasedAuthorization.repositories;

import com.codingsrv.RoleBasedAuthorization.entities.Session;
import com.codingsrv.RoleBasedAuthorization.entities.User;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.List;
import java.util.Optional;

public interface SessionRepository extends JpaRepository<Session, Long> {
    List<Session> findByUser(User user);

    Optional<Session> findByRefreshToken(String refreshToken);
}
